packages/ember-metal/tests/computed_test.js

```diff
@@ -900,6 +900,16 @@ QUnit.test('is chainable', function() {
   ok(cp instanceof ComputedProperty);
 });
 
+QUnit.test('throws assertion if called over a CP with a setter', function() {
+  throws(function() {
```

Comment:
Needs to use expectAssertion (because prod build tests will not throw since asserts are removed from prod builds).